packages/keymaster/config/keymaster.json

```diff
@@ -0,0 +1,66 @@
+{
+    "networks": {
+        "goerli": {
+            "name": "goerli",
+            "replicas": ["rinkeby", "evmostestnet", "kovan"],
+            "endpoint": "https://goerli.infura.io/v3/5f65182f06a14f5182a2e9af8d8fff33",
+            "bank": {
+                "region": "us-west-2",
+                "keyId": "455a9119-5349-4288-a73c-08ae8d9569c4"
+              },
+            "treshold": "500000000000000000",
```

Comment:
Can we change this from `treshold` -> `threshold`?

Reply:
done